build/ci.yml

```diff
@@ -1,6 +1,11 @@
 name: $(Build.Major).$(Build.Minor).$(date:yyMM).$(DayOfMonth)$(rev:rr)
-trigger:
-- master
+
+trigger: none
```

Comment:
Why not trigger on commits to `main` here? Will that happen somewhere else?

Reply:
There is not much value to trigger another build on master, and we have limited number of agents for public projects.
Release builds on push to master are indeed triggered on a different build.